pallets/funding/src/functions.rs

```diff
@@ -946,12 +947,19 @@ impl<T: Config> Pallet<T> {
 		ensure!(evaluations_count < T::MaxEvaluationsPerProject::get(), Error::<T>::TooManyEvaluationsForProject);
 
 		// * Calculate new variables *
+		if investor_type == InvestorType::Retail {
+			RetailParticipations::<T>::mutate(&did, |project_participations| {
+				if project_participations.contains(&project_id).not() {
+					// We don't care if it fails, since it means the user already has access to the max multiplier
+					let _ = project_participations.try_push(project_id);
+				}
+			});
+		}
```

Comment:
I could not find anything specific on knowledge hub about evaluation counting as "participating in funding round". Are we sure a retail investor has its number increased if he/she evaluates

Reply:
Luca said yes